include/fi_util.h

```diff
@@ -174,6 +175,7 @@ struct util_ep {
 	uint64_t		caps;
 	uint64_t		flags;
 	ofi_ep_progress_func	progress;
+	struct util_cmap	*cmap;
```

Comment:
Each endpoint can have a different transport address and so the connections would be specific to it.